rblib/tests/format_test.rb

```diff
@@ -59,7 +66,7 @@ def test_make_wrapped_urls_in_angle_brackets_clickable
 30 60 60 or 01625 545745 if you would prefer to call a national rate number,
 quoting your case reference number. You may also find some useful
 information on our website at
-&lt;<a href=\"https://web.nhs.net/owa/redir.aspx?C=25a8af7e66054d62a435313f7f3d4694&amp;URL=http%3a%2f%2fwww.ico.gov.uk%2f\">https://web.nhs.net/owa/redir.aspx?C=25a8af7e66054d62a435313f7f3d4694&amp;URL=http%3a%2f%2fwww.ico.gov.uk%2f</a>&gt; <a href=\"http://www.ico.gov.uk\">www.ico.gov.uk</a>."""
+&lt;<a href=\"http://web.nhs.net/owa/redir.aspx?C=25a8af7e66054d62a435313f7f3d4694&amp;URL=http%3a%2f%2fwww.ico.gov.uk%2f\">http://web.nhs.net/owa/redir.aspx?C=25a8af7e66054d62a435313f7f3d4694&amp;URL=http%3a%2f%2fwww.ico.gov.uk%2f</a>&gt; <a href=\"http://www.ico.gov.uk\">www.ico.gov.uk</a>."""
```

Comment:
it's an extra check so that the 2 links are different, tempted to break this one out into its own spec so it's clearer

if they're both https links, it's possible to pass the test while accidentally using the match from the first link for both substitutions